collection/Emperror; Emperror's New Groove.json

```diff
@@ -2273,16 +1983,31 @@
 			"classes": {
 				"fromClassList": [
 					{
-						"name": "Wizard",
+						"name": "Cleric",
+						"source": "PHB"
+					},
+					{
+						"name": "Paladin",
 						"source": "PHB"
 					}
+				],
+				"fromSubclass": [
+					{
+						"class": {
+							"name": "Warlock",
+							"source": "PHB"
+						},
+						"subclass": {
+							"name": "Celestial",
+							"shortName": "Celestial",
+							"source": "XGE"
+						}
+					}
 				]
 			},
 			"entries": [
 				"By means of a divine chant, the caster brings special favor upon themselves and their allies, and causes harm to their enemies. When this spell is cast, the caster begins chanting. The following effects take place while the chant is ongoing:",
-				"You and creatures of your choice within the spell's area gain a +1 bonus on saving throws, skill checks, and attack and damage rolls.",
-				"Creatures of your choice within the spell's area suffer a -1 penalty on saving throws, skill checks, and attack and damage rolls.",
-				"While you are chanting you cannot speak normally or cast spells with verbal components.  Additionally, your speed is halved while chanting.",
+				"<ul><li> You and creatures of your choice within the spell's area gain a +1 bonus on saving throws, skill checks, and attack and damage rolls.</li><li>Creatures of your choice within the spell's area suffer a -1 penalty on saving throws, skill checks, and attack and damage rolls.</li><li>While you are chanting you cannot speak normally or cast spells with verbal components.  Additionally, your speed is halved while chanting.</li></ul>",
```

Comment:
@TheGiddyLimit should these not be getting caught by the html tests?

Reply:
there's no test, it just gets clobbered

Reply:
Oh I see in the cleaning script. 
It might be nice to inform the user that it's going to occur. Can actions warn but not error?

Reply:
not afaik, and logging warnings without erroring is pretty useless

it'll now just fail instead :^)